app/Http/Controllers/PersonController.php

```diff
@@ -1,47 +1,23 @@
-<?php namespace Helix\Http\Controllers;
+<?php
 
-use Helix\Http\Controllers\Controller;
-use Helix\Http\Requests\Project\Create\StepOneRequest;
+declare(strict_types=1);
 
-use Helix\Models\Interest;
-use Helix\Models\Invitation;
-use Helix\Models\NemoMembership;
-use Helix\Models\Person;
+namespace Helix\Http\Controllers;
 
+use Auth;
+use Helix\Models\Invitation;
 use Helix\Models\Project;
-use Helix\Models\ProjectPolicy;
-use Helix\Models\Role;
-use Helix\Models\Research;
-use Helix\Models\Academic;
-use Helix\Models\Personal;
-use Helix\Models\Purpose;
-use Helix\Models\Attribute;
-use Helix\Events\Individual\IndividualAddAcademicInterests;
-use Helix\Events\Individual\IndividualAddPersonalInterests;
-use Helix\Events\Individual\IndividualAddInterests;
-
-use Illuminate\Database\Eloquent\ModelNotFoundException;
 use Illuminate\Support\Collection;
-use Searchy;
```

Comment:
?

Reply:
Don't remember removing this, but I will re-add it.